modules/extensions-xml-support/src/test/resources/compatibility/modules/module-xsd-custom-types.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module name="module-xsd-custom-types"
+        minMuleVersion="4.0.0"
+        xmlns="http://www.mulesoft.org/schema/mule/module"
+        xmlns:mule="http://www.mulesoft.org/schema/mule/core"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="
+           http://www.mulesoft.org/schema/mule/module http://www.mulesoft.org/schema/mule/module/current/mule-module.xsd
+           http://www.mulesoft.org/schema/mule/core http://www.mulesoft.org/schema/mule/core/current/mule.xsd">
+
+    <operation name="operation-with-custom-types">
+        <parameters>
+            <parameter name="value" type="XsdType1"/>
+            <!--extra parameter without targetNamespace in the schema under XsdType3-->
+            <parameter name="value2" type="XsdType3"/>
+        </parameters>
+        <body>
+            <mule:set-payload value="#[parameters.value]"/>
+        </body>
+        <output type="XsdType2"/>
```

Comment:
we do: https://www.mulesoft.org/jira/browse/MULE-12928
It will be an extra element after `<output>`, optional, called `<output-attribute type="bleh"/>` (by default, will be void)